### DISCUSSION TITLE
Is there an easy way to filter which modules are included in the build?

### DISCUSSION THREAD

**Discussion by mpicard** [DISTRHO/Cardinal#839] [Q&A]
Hey folks I would like to create my own build with a small subset of plugins, I only use a handful of modules.

**Comment by dromer** [comment]
"easy" is relative. You will have to modify `plugins/plugins.cpp` and probably a few other bits.

Perhaps see how this is done for CardinalMini which is essentially exactly that: a custom build with only a subset of modules included.
